### PR TITLE
Display info message when trying to delete a key signature that can't be deleted

### PR DESCRIPTION
**Pull request by mike-spa**
Resolves: #17542

Namely:

- Initial key signature of the score
- Key signature determined by an instrument change

### CODE REVIEW

**Review by oktophonie — APPROVED** [review]

**Comment by bkunda** [comment]
Tested on Win. All looks good.
Only one thing to nitpick:
Screenshot 2023-06-27 132003
The full stop at the end of the dialog header shouldn't be there
Once this is fixed, I'm happy!

**Comment by mike-spa** [comment]
> The full stop at the end of the dialog header shouldn't be there

Done! 👍

File: src/notation/internal/notationinteraction.cpp [hunk 1]
```diff
@@ -364,6 +364,10 @@ void NotationInteraction::checkAndShowMScoreError() const
     case MsError::CORRUPTED_MEASURE:
         title = trc("notation", "Cannot change time signature in front of a corrupted measure");
         break;
+    case MsError::CANNOT_REMOVE_KEY_SIG:
+        title = trc("notation", "This key signature cannot be deleted");
+        message = trc("notation", "Please replace it with a key signature from the palettes instead.");
```
[the remark this answers is not in the document]
Comment (Jojo-Schmitz) [reply src/notation/internal/notationinteraction.cpp:369]: Might be an option for a 41.0 port?

[the remark this answers is not in the document]
Comment (cbjeukendrup) [reply src/notation/internal/notationinteraction.cpp:369]: Transifex auto-fills the translation if the same string already exists but in a different context. So no need to use the same context

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**